runtime.md

```diff
@@ -34,22 +34,20 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is started to when it is stopped or crashes unexpectedly.
+
+* OCI compliant runtime is invoked by passing the bundle path as argument.
+* The container's runtime environment is created.
+* The prestart hooks are invoked in the host environment by the runtime.
+  If any prestart hook fails, then the container is killed and an error code is returned.
+* The user specified process is executed in the container.
+* The poststart hooks are invoked in the host environment by the runtime.
+  If any poststart hook fails, then the container is killed and an error code is returned.
+* Additional actions such as pausing the container, checkpointing the container, resuming the container or signaling the container may be performed using the runtime interface.
+  The container could also error out or crash.
+* The container is destroyed by undoing the steps performed during create phase if required.
+* The poststop hooks are invoked in the host environment and errors, if any, are logged.
```

Comment:
Since all hooks are invoked in the host environment, I'd just add that documentation to the hooks section (e.g. [my attempt here][1]).  Documenting that in *both* locations can't hurt ;), but having that documented only in the lifecycle section seems odd.

[1]: https://github.com/opencontainers/specs/pull/207/files#diff-695bac0ce528e5c6765a6666fcccc425R15